src/Compatibility/Core/tests/Android/ObservrableItemsSourceTests.cs

```diff
@@ -21,13 +22,13 @@ public async Task ObservableSourceItemsCountConsistent()
 			source.Add(1);
 			source.Add(2);
 
-			var ois = ItemsSourceFactory.Create(source, new MockCollectionChangedNotifier());
+			var ois = ItemsSourceFactory.Create(source, Application.Current, new MockCollectionChangedNotifier());
```

Comment:
This is weird, but this is exactly what Device.InvokeOnMainThread did. And in the case of a test, the "container" is the app. So... yeah?